tests/test_widgets.py

```diff
@@ -1791,5 +1792,45 @@ def testWidgetBadResult(self, banner_mock):
         self.presenter.movePositioner.assert_not_called()
 
 
+class TestTomographyTIFFLoader(unittest.TestCase):
+    @mock.patch("sscanss.app.window.presenter.MainWindowModel", autospec=True)
+    def setUp(self, model_mock):
+        self.view = TestView()
+        self.model_mock = model_mock
+        self.model_mock.return_value.instruments = [dummy]
+        self.model_mock.return_value.instrument.positioning_stack.name = dummy
```

Comment:
`self.model_mock.return_value.instrument.positioning_stack.name = dummy` this line is not needed please remove